index.html

```diff
@@ -1049,11 +1057,6 @@
 party, you expect them to have different discussion topics with you than they
 usually would, and possibly even to pretend they don't know you.
 
-Note that a user agent may not always be powerful enough to prevent [=cross-context
-recognition=]. For example, if two or more services [=inappropriately=]
-require an identifying piece of information (eg. an email address) in order
-to use the services.
```

Comment:
The comment about user agent power didn't really fit in the "recognition types" section, and I moved the "identifying piece of information" up to where we say what cross-context recognition is appropriate.